src/resources/postcss/tribe-events-full.pcss

```diff
@@ -2090,52 +2075,29 @@ input[name*='tribe-bar-']:-moz-placeholder {
 }
 
 /* Events Bar Icons */
-.tribe-select2-results-views span[class^='tribe-icon-'],
-.tribe-bar-views-list span[class^='tribe-icon-'],
-.tribe-bar-view-list a[class*='tribe-icon-'] {
```

Comment:
Here and below I'm removing a number of selectors for different versions of the view selector that no longer appear to be in play. If I'm mistaken, just let me know.

Reply:
I'm not really sure about this, perhaps @bordoni knows better.

Reply:
@bordoni Any input here? I just want to make sure I'm not neglecting a select2 style view selector that I should know about.